README.md

```diff
@@ -257,6 +257,6 @@ If you have a contribution in mind, please check out our [Contribution Guide](./
 
 <h2 id="license"> License </h2>
 
-Licensed under <a href="LICENSE">Apache 2.0 license</a>.
+Licensed under <a href="LICENSE">GNU General Public License v3.0</a>.
```

Comment:
You will need to update the readme badge as well. 

Use this:
```
[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
```

Instead of:
```
[![License Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](https://www.apache.org/licenses/LICENSE-2.0)
```

Reply:
Fixed